doc/_static/mpl.css

```diff
@@ -594,6 +594,10 @@ span.linkdescr {
     font-size: 90%;
 }
 
+span.highlighted {
```

Comment:
I don't understand why we need this; doesn't highlighting work already?

Reply:
Yes and no. Somehow it does not seem to work (anymore?). For that reason it probably makes sense to only let it apply to those codelines in the rc params for now.

Reply:
No actually, the highlighting was only for function signatures. So for example, https://matplotlib.org/1.4.1/api/axes_api.html?highlight=stem#matplotlib.axes.Axes.stem will show the stem line in yellow, but not every occurance of the word "stem". The signature highlighting probably got broken when intoducing the grey background for those. But I'm not sure how useful that is in general.